LoggerFinal/src/main/java/com/Logger/controllers/InventoryController.java

```diff
@@ -0,0 +1,64 @@
+package com.Logger.controllers;
+
+import com.Logger.domain.model.Inventory;
+import com.Logger.domain.model.Response;
+import com.Logger.utils.Response.InventoryResponse;
+import com.Logger.utils.RoomAvailabilityRequest;
+import com.Logger.services.InventoryServices;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequestMapping("/")
+@RestController
+public class InventoryController {
+
+    @Autowired
+    InventoryServices inventoryServices;
+
+    @RequestMapping(value = "/deleteInventoryDemo", method = RequestMethod.POST)
+    public String deleteDemo() {
+
+        return inventoryServices.deleteDemo();
+    }
+
+    /*@RequestMapping(value = "/api/logger/cm/inventory", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
+    public String saveInventory(@RequestBody List<Inventory> inventory) {
+        return inventoryServices.saveInventory(inventory);
+    }*/
+
+    @RequestMapping(value = "/api/logger/cm/inventory", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
/api/logger/ seems common across all controller request mapping functions. Please move it to RequestMapping defined above controller class